.pre-commit-config.yaml

```diff
@@ -45,14 +45,12 @@ repos:
           - --autofix
 
   - repo: https://github.com/PyCQA/flake8
-    rev: 7.1.0
+    rev: 7.1.1
     hooks:
       - id: flake8
         additional_dependencies: [
-          flake8-blind-except,
```

Comment:
Too re-enable, we'll probably need help from @will-moore on knowing what exceptions can be thrown.